apps/vr-tests-react-components/src/stories/Badge/BadgeSize.stories.tsx

```diff
@@ -159,3 +159,23 @@ export const SizeExtraLarge = () => {
 SizeExtraLarge.storyName = 'size: extra-large';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27489&commitId=94a8b1d8ded7780da8819e3fd9c9d4f179e0883b&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1929 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.normal.chromium.png|2| Changed|
|Badge Converged.width constrained.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>